src/FixedSizeArrays.jl

```diff
@@ -111,20 +111,20 @@ macro fixed_vector(name, parent)
         # Array constructor
         @inline function (::Type{$(name){S}})(x::AbstractVector{T}) where {S, T}
             @assert S <= length(x)
-            $(name){S, T}(ntuple(i-> x[i], Val{S}))
```

Comment:
`Val(S)` is a bit more readable (that constructor is `@pure`, so there should not be any type inference issues).